doc/manual/manual.tex

```diff
@@ -205,27 +205,30 @@ \chapter{Using \LaTeXML}\label{usage}
 \end{description}
 \noindent The usage of these commands can be as simple as
 \begin{lstlisting}[style=shell]
-latexml doc.tex | latexmlpost --dest=doc.html -
+latexmlc doc.tex --dest=doc.html
```

Comment:
I certainly like the command, but is this a good idea to change when latexmlc itself isn't officially documented in the [executables section](https://dlmf.nist.gov/LaTeXML/manual/commands/)? Or is that part of the plan?

Reply:
too late :>